src/system/RoutesModule/Resources/views/Route/view.tpl

```diff
@@ -8,39 +8,45 @@
 {if count($items) > 1}
 {pageaddvar value='jquery-ui' name='javascript'}
 <script type="text/javascript">
-    jQuery(function(){
-        // Return a helper with preserved width of cells
-        var fixHelper = function(e, ui) {
-            ui.children().each(function() {
-                jQuery(this).css({width: jQuery(this).width()});
-            });
-            return ui;
-        };
+    /* <![CDATA[ */
+    ( function() {
```

Comment:
See, I was confused why you wrapped the code. Because commonly, I wrap it to be able to use `$` as `jQuery` inside it.

IANAJSPAWBKAFT (I Am Not A JavaScript Programmer As Well But Know A Few Tricks)